AnnoMate/AppComponents/PhylogicComponents.py

```diff
@@ -132,19 +135,23 @@ def gen_ccf_plot(df, idx, time_scaled, samples_df):
         scatter_x = 'order'
         rect_x = 6
 
-    treatments_df = pd.read_csv(df.loc[idx, 'treatments_fn'], sep='\t', comment='#')
-    treatments_in_frame_df = treatments_df[(treatments_df['stop_date_dfd'] >= int(timing_data[samples_in_order[0]])) &
+    if 'treatments_fn' in list(df):
+        treatments_df = pd.read_csv(df.loc[idx, 'treatments_fn'], sep='\t', comment='#')
+        treatments_in_frame_df = treatments_df[(treatments_df['stop_date_dfd'] >= int(timing_data[samples_in_order[0]])) &
                                            (treatments_df['start_date_dfd'] <= int(timing_data[samples_in_order[-1]]))]
 
     # get mutation counts
     mut_ccfs = pd.read_csv(df.loc[idx, 'maf_fn'], sep='\t')
+    participant_id = mut_ccfs.columns[mut_ccfs.columns.isin(['Patient_ID', 'Participant_ID', 'participant_id'])][0]
```

Comment:
I think these columns should all be able to be specified directly by the user, too